ui/components/app/gas-details-item/gas-details-item.js

```diff
@@ -90,8 +90,8 @@ const GasDetailsItem = ({ userAcknowledgedGasMissing = false }) => {
       }
       subTitle={
         <GasTiming
-          maxPriorityFeePerGas={maxPriorityFeePerGas}
-          maxFeePerGas={maxFeePerGas}
+          maxPriorityFeePerGas={maxPriorityFeePerGas.toString()}
```

Comment:
LGTM! Heads up, we might want to do future work on these two props, `maxPriorityFeePerGas` and `maxFeePerGas`, to keep data type passed around our app consistent, as strings. Some details in this ticket https://github.com/MetaMask/metamask-extension/pull/14665